### DISCUSSION TITLE
assertEq is showing error when written in the test file

### DISCUSSION THREAD

**Discussion by sardarahmedkhan505** [Cyfrin/foundry-full-course-cu#240] [Q&A] [answered]
### Undeclared identifier. Did you mean "assert"? this is the actual error
> This is my actual code >
`// SPDX-License-Identifier: MIT
pragma solidity ^0.8.18;
import {Test, console} from "forge-std/Test.sol";
import {FundMe} from "../src/FundMe.sol";

contract FundMeTest {
    FundMe fundme;

    function setup() external {
        fundme = new FundMe();
    }

    function checkMminimumDollar() public {
        assertEq(fundme.MINIMUM_USD(), 5e18);
    }
}
`

**Comment by Shiva-Sai-ssb** [comment, marked as answer]
Hey @sardarahmedkhan505 , you forgot to inherit the "Test" contract on the "FundMeTest" contract declaration. It should be like this:

```solidity
contract FundMeTest is Test {

}
```

**Reply by sardarahmedkhan505** [reply]
Thanks solved my problem, I was forgotten the inheritance from Test Interface.
